packages/x-internals/package.json

```
@@ -45,6 +45,7 @@
     "@mui/utils": "^7.0.0 || ^7.0.0-beta"
   },
   "peerDependencies": {
+    "@mui/system": "^5.15.14 || ^6.0.0 || ^7.0.0-alpha",
```

```suggestion
    "@mui/system": "^5.15.14 || ^6.0.0 || ^7.0.0",
```
Since v7 stable is already available